docs/dev/agent_integration.md

```diff
@@ -21,6 +22,52 @@ The required Datadog Agent integration development tools include:
 - [Docker][4] to run the full test suite.
 - The git [command-line][5] or [GitHub desktop client][19].
 
+You can list an Agent integration for a price in the Datadog Marketplace, or list it out-of-the-box on the Integrations page.
+
+{{< tabs >}}
+{{% tab "Marketplace Integrations" %}}
```

Comment:
Can we change this to something clearer? Like "Build Marketplace Agent Integration" or just "Build Marketplace Integration" if it needs to be shorter?

Same with the Out-of-the-box button